### ISSUE TITLE
zstd packages are not supported

### ISSUE THREAD

**Issue by scott323** [seblu/agetpkg#5]
As I only need download functionality, I've fixed this temporarily with:

```
< PKG_EXT = ".pkg.tar.xz"
> PKG_EXT = ".pkg.tar.zst"
```

**Comment by seblu** [comment]
Fixed in e8f14d52dd2ea4b8e45a88a86a51cd0c83c91214.

There is some use cases (e.g -v -l) where listing is slow. I need to look at it before releasing.

**Comment by seblu** [comment]
So, the latency come from several HTTP redirection when `archive.archlinux.org` redirect us to `archive.org` because there is no local version of the file.

````
$ time curl --head -L https://archive.archlinux.org/packages/.all/nodejs-11.7.0-1-x86_64.pkg.tar.lz4
HTTP/2 302 
server: nginx/1.18.0
date: Thu, 10 Sep 2020 04:15:14 GMT
content-type: text/html
content-length: 145
location: https://archive.org/download/archlinux_pkg_nodejs/nodejs-11.7.0-1-x86_64.pkg.tar.lz4
strict-transport-security: max-age=31536000; includeSubdomains; preload

HTTP/1.1 302 Found
Server: nginx/1.16.1 (Ubuntu)
Date: Thu, 10 Sep 2020 04:15:15 GMT
Content-Type: text/html; charset=UTF-8
Connection: keep-alive
Accept-Ranges: bytes
Location: https://ia903102.us.archive.org/1/items/archlinux_pkg_nodejs/nodejs-11.7.0-1-x86_64.pkg.tar.lz4
Strict-Transport-Security: max-age=15724800

HTTP/1.1 404 Not Found
Server: nginx/1.16.1 (Ubuntu)
Date: Thu, 10 Sep 2020 04:15:17 GMT
Content-Type: text/html; charset=UTF-8
Connection: keep-alive
Set-Cookie: PHPSESSID=8dqb1ujr6fa5vl4m9sa3v2pet1; path=/; domain=.archive.org
Expires: Thu, 19 Nov 1981 08:52:00 GMT
Cache-Control: no-store, no-cache, must-revalidate
Pragma: no-cache
Strict-Transport-Security: max-age=15724800

curl --head -L   0,03s user 0,00s system 0% cpu 3,244 total
```

About 3s per package, when you are requesting hundred of packages information...
In a regular use, without the **-v** option, testing the file extension is limited to those we download, which often a limited number.

A long term fix, would be to upgrade the index to include the file extension.
